### ISSUE TITLE
Intempestive player disconnections on some 1.12.4 servers

### ISSUE THREAD

**Issue by Legonzaur** [R2Northstar/Northstar#448]
# Bug

The error players see is `Disconnect: Sent too many stringcmd commands.`

Since update of Fvnkhead's servers to [`1.12.4`](https://github.com/R2Northstar/Northstar/releases/tag/v1.12.4), players have been experiencing intermpestive disconnections from gameservers. When I talked about it on Discord, it seemed that some other servers are also concerned, in particular Neinguar's servers.

# Steps to reproduce
As far as I experienced, the disconnections occur either shortly after spawning or after getting killed.

# Logs
Below are some logs I collected from myself or other players when this issue arised
[nslog2023-04-06 15-58-56.txt](https://github.com/R2Northstar/NorthstarMods/files/11172758/nslog2023-04-06.15-58-56.txt)
[nslog2023-04-06 15-58-56 (2).txt](https://github.com/R2Northstar/NorthstarMods/files/11172759/nslog2023-04-06.15-58-56.2.txt)
[nslog2023-04-06 15-58-56 (3).txt](https://github.com/R2Northstar/NorthstarMods/files/11172760/nslog2023-04-06.15-58-56.3.txt)
[nslog2023-04-06_13-47-15.txt](https://github.com/R2Northstar/NorthstarMods/files/11172764/nslog2023-04-06_13-47-15.txt)
[nslog2023-04-06_11-41-18.txt](https://github.com/R2Northstar/NorthstarMods/files/11172765/nslog2023-04-06_11-41-18.txt)
[nslog2023-04-06_10-55-11.txt](https://github.com/R2Northstar/NorthstarMods/files/11172766/nslog2023-04-06_10-55-11.txt)

**Comment by Legonzaur** [comment]
Sometimes, after disconnecting or being disconnected, players encounter an issue where they can't rejoin the server anymore.
The error players see is `Connection to server timed out` whenever they try to join the server they got disconnected from.
Joining other servers works fine.

This issue can last from 20 minutes to around 1 hour. Afterwards, the player can rejoin the server again

While this can happen when being kicked out with the message `Disconnect: Sent too many stringcmd commands.`, this can also happen when retrying to disconnect and reconnect from the server because of #449 

I don't know if I create file another issue or use this one, but those two problems might be related.

I had some results trying to continue to play by switching between 4G and ADSL whenever the server didn't wanted me to join again, until I ultimately was unable to join with both internet connections.

![image](https://user-images.githubusercontent.com/34353603/230563813-cb0c8ffb-988f-4d8e-b49f-e40ebc0d7684.png)

**Comment by ASpoonPlaysGames** [comment]
Are these logs client logs or server logs?

**Comment by Legonzaur** [comment]
After a rollback to 1.12.3 this issue as well as #449 and #450 are gone

**Comment by Legonzaur** [comment]
After a server rollback from 1.12.4 to 1.12.3 this issue as well as #449 and #450 are gone

**Comment by pg9182** [comment]
https://github.com/R2Northstar/NorthstarLauncher/pull/411 is probably the cause then.

**Comment by JMM889901** [comment]
This also currently occurs to me while testing with the safeIO build in private match, often when opening menus(even the settings menu at one point i think) and switching between loadouts, it's possible this is due to framework but it doesn't seem to have any correlation to its clientcommand usage, unless runclientscript/runuiscript causes it  
log is attached, but not sure how much use it will be, got the issue about 7 times
[nslog2023-04-05 16-53-54.txt](https://github.com/R2Northstar/Northstar/files/11181764/nslog2023-04-05.16-53-54.txt)

**Comment by Legonzaur** [comment]
Problem resolved after #447 and/or `v1.12.5`
